### PR TITLE
10.0 mig account mass reconcile transaction ref

### PR DESCRIPTION
**Pull request by damdam-s**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check Travis. Maybe a missing dependency in oca_repositories.txt?

**Comment by damdam-s** [comment]
still the same error linked to `account_mass_reconcile` test :cry:

**Comment by pedrobaeza** [comment]
Agh, I remember... Let's chat in #204

**Comment by fclementic2c** [comment]
functional test Ok
:+1:

**Review by grindtildeath — CHANGES_REQUESTED** [review]
:warning: 
There is a breaking typo :

 - https://github.com/OCA/account-reconcile/pull/207/files#diff-d89dd1cfc093f09b910f0fce7ad31e0bR18
 - https://github.com/OCA/account-reconcile/pull/207/files#diff-9fec5a69c0fff00d745de1a9fb012cbaR35

**Review by grindtildeath — APPROVED** [review]
Now it works :+1:

File: account_move_base_import/views/account_move_view.xml [hunk 1]
```diff
@@ -64,4 +64,22 @@
 
     <menuitem string="Move Completion Rule" action="action_move_completion_rule_tree"
               id="menu_action_move_completion_rule_tree_menu" parent="account.account_management_menu"/>
+
+    <record id="view_account_move_filter" model="ir.ui.view">
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_account_move_filter"/>
+        <field name="arch" type="xml">
+            <separator position="after">
+                <filter
```
Comment (Tonow-c2c) [inline account_move_base_import/views/account_move_view.xml:73]: you should add that for migration ?

Comment (grindtildeath) [reply account_move_base_import/views/account_move_view.xml:73]: Good catch ! @damdam-s can you drop the related commit ?

**Review by simahawk — APPROVED** [review]
eyeball review: LG

**Comment by grindtildeath** [comment]
Since @damdam-s is too busy, I opened #222 and this can be closed.

**Pull request closed without merging**